### PR TITLE
... and now something completely SPID

### PR DESCRIPTION
**Pull request by peppelinux**
Added eID resources to public services developers and administrators
    
* feat: added SPID shortcut
* feat: added CIE shortcut
* feat: added eIDAS shortcut

Revamping SPID
    
* feat: some brand new contents

### CODE REVIEW

File: src/configuration/tree.yaml [hunk 1]
```diff
@@ -33,6 +33,24 @@ root:
               Scopri gli allegati tecnici delle Linee guida per lo sviluppo e la manutenzione del software
               <https://docs.italia.it/italia/developers-italia/lg-acquisizione-e-riuso-software-per-pa-docs/it/stabile/index.html>
 
+          - label: "Vorrei integrare la mia piattaforma con il sistema di Single SignOn Nazionale ed Europeo"
+            content:
+
+            - label: "SPID - Sistema Pubblico di Identità Digitale"
+              content: |
+                Scopri la documentazione e tutte le risorse dedicate a SPID
+                <https://developers.italia.it/it/spid/>
```
Comment (bfabio) [inline src/configuration/tree.yaml:42]: All the links should follow [this format](https://github.com/italia/wizard-italia/blob/main/src/configuration/tree.yaml#L20-L24), using `target="_blank"`, an `aria-label` explaining the link will lead to an external site and preferably a description with an icon signaling the external link.

File: src/configuration/tree.yaml [hunk 2]
```diff
@@ -159,17 +177,50 @@ root:
...
+                  Qui trovi le risorse di esempio e i progetti OpenSource dedicati a SPID, oltre alla documentazione tecnica e altre risorse:
+                  <https://developers.italia.it/it/spid>
+
+              - label: "Qualcosa di utile per iniziare"
+                question: "Hai voglia di metterti in gioco?"
+                content:
+
+                    - label: "Contribuire sulla validazione dei test"
+                      content: |
+                       <https://github.com/italia/spid-sp-shibboleth/issues/9>
```
Comment (bfabio) [inline src/configuration/tree.yaml:196]: I'm not sure we want this resolution and to link specific issues, they'll need to be maintained and removed from here when they are solved.

Maybe we can link to a GitHub search with specific labels? 

Wdyt @astagi @ruphy?

**Review by bfabio — CHANGES_REQUESTED** [review]

File: src/configuration/tree.yaml (hunk 1 above)
Comment (peppelinux) [reply src/configuration/tree.yaml]: [whatch this](https://www.youtube.com/watch?v=z6jNekFYAyg)
https://github.com/italia/wizard-italia/pull/11/commits/d54ea032b675af7e20ece4861b7d0f93178c1931

File: src/configuration/tree.yaml (hunk 2 above)
Comment (peppelinux) [reply src/configuration/tree.yaml]: @bfabio removed, don't mind

**Pull request merged**